hack/test-integration.sh

```diff
@@ -26,6 +26,7 @@ function cleanup()
 package="${OS_TEST_PACKAGE:-test/integration}"
 tags="${OS_TEST_TAGS:-integration no-docker}"
 
+export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
```

Comment:
why not a tiny go program we can capture the output of, e.g. http://play.golang.org/p/E_mkDidnca